charts/spdk-csi/latest/spdk-csi/values.yaml

```diff
@@ -136,10 +148,14 @@ cachingnode:
 storagenode:
   create: false
   ifname: eth0
+  distr_ndcs: "1"
+  distr_npcs: "1"
   spdkImage:
   maxLvol: 10
```

Comment:
Should we not use abbreviations but "maxLogicalVolumes" and "maxSnapshots"?

I think there is ne usability benefit of abbreviating it.

Reply:
Do we need to update the parameters in the README for this?

Reply:
Yes, I think we do